tests/BUILD

```diff
@@ -88,10 +88,30 @@ cc_test(
     ],
 )
 
+cc_library(
```

Comment:
I would recommend adding `testonly = 1,` to this rule (see [bazel docs](https://docs.bazel.build/versions/master/be/common-definitions.html) for testonly usage).